### PR TITLE
Remove Sequel models

### PR DESCRIPTION
**Pull request by gonzaloriestra**
Remove some Sequel models and always use ActiveRecord instead:
- `Like`
- `Tag`
- `OauthNonce` (unused)

### CODE REVIEW

File: app/models/table.rb [hunk 1]
```diff
@@ -442,7 +442,7 @@ def handle_creation_error(e)
   end
 
   def after_destroy
-    Tag.filter(user_id: user_id, table_id: id).delete
+    Carto::Tag.where(user_id: user_id, table_id: id).each(&:destroy)
```
Comment (alrocar) [inline app/models/table.rb]: Not sure about if we should use `destroy!` in this case. If we don't do it we could have some zombie `tags`

File: app/models/table.rb [hunk 2]
```diff
@@ -1469,10 +1469,10 @@ def update_the_geom!(attributes, primary_key)
 
   def manage_tags
     if @user_table[:tags].blank?
-      Tag.filter(:user_id => user_id, :table_id => id).delete
+      Carto::Tag.where(user_id: user_id, table_id: id).each(&:destroy)
```
Comment (alrocar) [inline app/models/table.rb]: Same here

File: app/models/table.rb (hunk 1 above)
Comment (gonzaloriestra) [reply app/models/table.rb]: In Sequel, [`delete`](https://www.rubydoc.info/github/jeremyevans/sequel/Sequel/Model/InstanceMethods#delete-instance_method) is equivalent to AR's `destroy`, while `destroy` is similar to AR's `destroy!`.

I'm not sure about this, so I'd prefer to keep the behavior for now, as this is already a quite big refactor. What do you think?

Comment (alrocar) [reply app/models/table.rb]: LGTM

**Comment by amiedes** [comment]
Acceptance OK, just a couple of comments:

- I think the `NEWS.md` file must be updated in order for the tests to pass
- For a while, when liking datasets from https://amiedes-acc-sequel-models.carto-staging.com/dashboard/datasets (`amiedes-acc-sequel-models` / `amiedes-acc-sequel-models123`) the star icon was not refreshing it's color (the HTTP request was OK, but I had to refresh the page in order to see the changes).

  But misteriously after some more browsing it was working ok again and couldn't replicate it anymore. In any case, I don't think that's related to your refactor.

The cases I tested were:

- Adding and deleting tags for both datasets and maps
- Listing the items that contain a tag
- Liking / unliking datasets and maps

**Comment by gonzaloriestra** [comment]
@amiedes Thanks!

Regarding the News file, tests fail as a reminder to update it. But in this case, as this is just a refactor, it's better to avoid adding noise there. We can merge anyway.

About the star refreshing issue, it doesn't look related. But if you see that again, we can create a new issue.

**Pull request merged**